packages/moonstone/Picker/SpottablePicker.js

```diff
@@ -11,18 +11,21 @@ const SpottablePicker = hoc(null, (config, Wrapped) => {
 
 		static propTypes = {
 			joined: PropTypes.bool,
+			onSpotlightDisappear: PropTypes.func,
 			onSpotlightDown: PropTypes.func,
 			onSpotlightLeft: PropTypes.func,
 			onSpotlightRight: PropTypes.func,
 			onSpotlightUp: PropTypes.func
 		}
 
 		render () {
-			const {onSpotlightDown, onSpotlightLeft, onSpotlightRight, onSpotlightUp, ...rest} = this.props;
+			const {onSpotlightDisappear, onSpotlightDown, onSpotlightLeft, onSpotlightRight, onSpotlightUp, ...rest} = this.props;
```

Comment:
I don't think it makes sense to forward this on to the buttons. The only case in which `onSpotlightDisappear` fires is on unmount now. In the case of a joined picker, this will be handled correctly by Spottable. In the case of separate picker buttons, we should generate the `onSpotlightDisappear` at the picker boundary rather than emitting two events (one from each button). We likely need similar logic for IncrementSlider as well.